### PR TITLE
GP2-924: Use video thumbnails

### PR DESCRIPTION
**Pull request by stevejalim**
**DO NOT MERGE UNTIL MONDAY**

This changeset supports using the thumbnail image uploaded in the CMS as the `poster` attribute of our `<video>` elements

 - [X] Ticket exists in Jira  https://uktrade.atlassian.net/browse/GP2-924 
 - [X] Jira ticket has the correct status.
 - [X] [Changelog](CHANGELOG.md) entry added.

 - [X] Includes screenshot(s) - see video attached to https://uktrade.atlassian.net/browse/GP2-924 
 
 - [ ] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: core/templatetags/personalised_blocks.py [hunk 1]
```diff
@@ -19,11 +25,12 @@ def render_video_block(block):
...
     if not block:
         return ''
-    sources = format_html_join('\n', '<source{0}>', [[flatatt(source)] for source in block['video'].sources])
+    video = block['video']
+    sources = format_html_join('\n', '<source{0}>', [[flatatt(source)] for source in video.sources])
     return format_html(
         f"""
                 <div>
-                    <video width="{block['width']}" height="{block['height']}" controls>
+                    <video {_get_poster_attribute(video)}width="{block['width']}" height="{block['height']}" controls>
```
Comment (webbyfox) [inline core/templatetags/personalised_blocks.py:33]: why not space here between attributes compare to space at function at L12?

**Review by webbyfox — COMMENTED** [review]
Looks good !! just a minor question...

File: core/templatetags/personalised_blocks.py (hunk 1 above)
Comment (stevejalim) [reply core/templatetags/personalised_blocks.py:33]: Not a massive reason but if `_get_poster_attribute(video)` doesn't find a thumbnail, it returns `''`, so we don't ned up with an extra space before in `<video width=...` (ie we don't end up with `<video  width=...``

**Review by webbyfox — APPROVED** [review]
LGTM

**Pull request merged**